src/components/Activities.jsx

```diff
@@ -5,7 +5,7 @@ import { addActivityType } from "../modules/destination.js";
 
 const Activities = props => {
   const [activityType, setActivityType] = useState(null);
-  const [actTimes, setActTimes] = useState(null);
+  const [activity_visits, setActivity_visits] = useState(null);
```

Comment:
activityVisits - but I got your back. Fixed in PR to be merged into this one.

Reply:
ok ok tack